app/export.go

```diff
@@ -172,9 +172,12 @@ func (app *OsmosisApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddr
 		counter++
 	}
 
-	iter.Close()
+	err := iter.Close()
+	if err != nil {
+		panic(err)
```

Comment:
nit: just to be consistent :) 


```suggestion
		log.Fatal(err)
```